src/app/MessageDef/WriteRequestMessage.h

```diff
@@ -40,6 +40,7 @@ enum class Tag : uint8_t
     kTimedRequest        = 1,
     kWriteRequests       = 2,
     kMoreChunkedMessages = 3,
+    kFabricFiltered      = 4,
```

Comment:
@erjiaqing looks like we miss FabricFiltered check feature in fabric write PR?

Reply:
Write are always fabric filtered, this should be removed.

Reply:
Will remove from spec rather than add to SDK:
https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/5235

Reply:
Already removed in [this](https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/5208/files#diff-269a68b9033e462f978c22a9fbfc4bccd403c58f201dadb6e4d8f59448b30913L1112) pending PR.